CollAction/Models/ApplicationUser.cs

```diff
@@ -11,6 +11,7 @@ public class ApplicationUser : IdentityUser
         public string FirstName { get; set; }
         [MaxLength(250)]
         public string LastName { get; set; }
+        public bool NewsletterSubscription { get; set; }
```

Comment:
I was under the impression that we need newsletter subscriptions for both registered users, in that case we'd want to store the subscriptions in a separate NewsletterSubscription table. Check with Daniella?